# Papers to read
https://plato.stanford.edu/archives/win2019/entries/truth-revision/
https://seop.illc.uva.nl/entries/models-science/


# Blogs to read
https://bioeticacomplexa.blogspot.com/2020/05/
https://bioeticacomplexa.blogspot.com/2020/07/
https://addshore.com/2021/05/wbstack-setting-changes-federated-properties-wikidata-entity-mapping-more/
https://www.kmjn.org/notes/academic_ancestors_wikidata.html
# Books to read
Huxley, J. S.: The new systematics (Oxford Univ. Press, Oxford 1940).

Lorhard, J. (1606). Diagraph of Metaphysic or Ontology.St. Gallen: Ogdoas scholastica Book 8. PublishedinSangalli.Trans.Sara L. Uckelman.

Popper, K. R. Objective Knowledge: An Evolutionary Approach. Oxford: Clarendon, 1975.

 Elton CS. 1927. Animal Ecology. https://doi.org/10.5962/bhl.title.7435 

Wagner GP. 2014. Homology, Genes, and Evolutionary Innovatoin.
Princeton, NJ: Princeton University Press

13.  Robinson  PN,  Bauer  S.  Introduction  to  biol-ontologies.  Boca  Raton,  FL:    CRC  Press, 2011
# Cell types
Q90348985
Q49169602
## No QID

## Other
Terminologia histologicainternational terms for human cytology and histologyFederative International Committee on Anatomical Terminology (FICAT)

http://dedalus.usp.br/F/8GPMJ42AQI65MDTA9R1XSB1XVXC1Y9AEYY3P4SU558H8FRN3XP-16162?func=item-global&doc_library=USP01&doc_number=002125570&year=&volume=&sub_library=FM
https://link.springer.com/article/10.2478%2Fs11756-019-00214-1

# Biocuration
Q106708178
Q94470608
Q52311286
Q97685936
Q89944322
Q89435784
Q65002314
Q91956367
Q97683003
Q59538811
Q98223660
Q94954393
Q98209024
Q56815361
Q59789431
Q33787495
Q54898134
Q94558276
Q28950350
Q53838636
Q90290716
Q90415774
Q92358532
Q56655129
Q46645138
Q91566046
Q98618031
Q91044884
Q38440697
Q58161139
Q64111054
Q102372282
Q58587002
Q101222862
Q49826285
Q58611639
Q47555736
Q56992710
Q37060118
Q93337500
Q58611634


## No QID

https://www.biogateway.eu/ssb/
Himmelstein,D. . https://thinklab.com/discussion/integrat ing-resources-with-disparate-licensing-into-an-open-network/ 107#1 (accessed Sep 16, 2016).
Lmn-2 interacts with Elf-2. On the meaning of common statements in biomedical literature
The Next Big Thing: From Hypermedia to Datuments
https://www.semanticscholar.org/paper/Ontology-%3A-Foundation-of-biomedical-big-data-and-Jie-Yongqun/e94c4a60a8e58daaf39952e06266d197a81ca24f

https://scite.ai/ 

Comments to the first nomenclature of human cytology: the description of cells and their ultrastructure in the Terminologia Histologica. Which important medical and biological terms are disputable or missing?

## Other

https://www.ebi.ac.uk/spot/zooma/


COELHO, KatiaCardoso. Aquisição de conhecimento especializado para construção de ontologias: um estudo no domínio das ciências da vida. 2012.174f. Dissertação (Mestradoem Ciência da Informação). Escola de Ciência da Informação, Universidade Federal de Minas Gerais, Belo Horizonte, 2012. Disponível em: http://www.bibliotecadigital.ufmg.br/dspace/bitstream/handle/1843/ECID-8XLQS4/disserta__o_katia_cardoso_coelho.pdf?sequence=2. Acesso em: ago.2014

Bio4j: a high-performance cloud-enabled graph-based data platform.
Search, access, and explore life sciencenanopublications on the Web
Micropublication: incentivizing community curation and placing unpublished data into the public domain

SCHRIML, Lynn M.; et al. Disease Ontology: a backbone for disease semantic integration. Nucleic Acids Research. v.40. 2012. Disponível em: http://nar.oxfordjournals.org/content/40/D1/D940.full.pdf?keytype=ref&ijkey=oOOVAj4zMzRpayq. Acesso em:21 set 2014

USCHOLD, Mike.Building ontologies: Towards a Unified Methodology (1996). Disponível em:http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.39.9075. Acesso em:15 fev 2013

The role of trust in knowledge

https://doi.org/10.1093/nar/gks1193

 Colin N. Dewey. 2017. “MetaSRA: Normalized HumanSample-Specific Metadata for the Sequence Read Archive.”Bioinformatics (Oxford, England)33 (18): 2914–23. https://doi.org/10.1093/bioinformatics/btx334.

 https://www.zotero.org/google-docs/?OzuVZW

“Minimum Information about a Microarray Experiment(MIAME)—toward Standards for Microarray Data.”Nature Genetics29 (4): 365–71.https://doi.org/10.1038/ng1201-365.

Galeota, Eugenia, and Mattia Pelizzola. 2017. “Ontology-Based Annotations and SemanticRelations in Large-Scale (Epi)Genomics Data.”Briefings in Bioinformatics18 (3): 403–12.https://doi.org/10.1093/bib/bbw036.

“ALE: Automated LabelExtraction from GEO Metadata.”BMC Bioinformatics18 (14): 509.https://doi.org/10.1186/s12859-017-1888-1.Hadley, Dexter, James Pan, Osama El-Sayed,

Jihad Aljabban, Imad Aljabban, Tej D. Azad,Mohamad O. Hadied, et al. 2017. “Precision Annotation of Digital Samples in NCBI’s GeneExpression Omnibus.”Scientific Data4 (1): 170125. https://doi.org/10.1038/sdata.2017.125.


“Ontology-AwareClassification of Tissue and Cell-Type Signals in Gene Expression Profiles across Platforms andTechnologies.”Bioinformatics29 (23): 3036–44. https://doi.org/10.1093/bioinformatics/btt529.


“Real-Time Tagging of Biomedical Entities.”BioRxiv, September, 078469. https://doi.org/10.1101/078469.Panahiazar, Maryam, Michel Dumontier, and Olivier Gevaert. 2017. 

“Predicting BiomedicalMetadata in CEDAR: A Study of Gene Expression Omnibus (GEO).”Journal of BiomedicalInformatics72 (August): 132–39. https://doi.org/10.1016/j.jbi.2017.06.017.


“Extraction and Analysis ofSignatures from the Gene Expression Omnibus by the Crowd.”Nature Communications7 (1):12846. https://doi.org/10.1038/ncomms12846.


# Top 100 papers
# Random
Q92575623
Q90696426
Q33535717
Q92575623

